modules/s3cfg.py

```diff
@@ -3054,6 +3054,12 @@ def get_req_requester_from_site(self):
         """
         return self.req.get("requester_from_site", False)
 
+    def get_req_task_list_field(self):
```

Comment:
Have a look at how other settings functions that show / hide list fields are named -- is there a convention?